fs/expose/dokan/__init__.py

```diff
@@ -89,22 +100,46 @@
 import logging
 logger = logging.getLogger("fs.expose.dokan")
 
+if six.PY3:
```

Comment:
Sorry... I didn't know that. I agree with your `from six.moves import range`.

Thanks for telling me this. Didn't bother checking the documentation of `six`.